frontend/src/routes/index.tsx

```diff
@@ -14,6 +21,141 @@ function RouteComponent() {
       <PageText $variant="ingress">
         This is an application for managing the settings of FMU projects.
       </PageText>
+      <Documentation />
     </>
   );
 }
+
+function Documentation() {
+  return (
+    <div
+      style={{
+        display: "flex",
+        justifyContent: "flex-start",
+        flexWrap: "wrap",
+        padding: "20px",
+        gap: "15px",
+      }}
+    >
+      <CardContainer>
+        <Card.Header>
+          <Card.HeaderTitle>
+            <Card.Content>
+              <FmuLogo src={fmuLogo} />
+            </Card.Content>
+            <Typography variant="h5">FMU Hub</Typography>
+            <Typography variant="body_short">
+              FMU Hub - the place for everything related to FMU. Expert or noob,
+              user or developer, skeptic or believer - the site is for you and
+              the rest of the amazing FMU community.
+            </Typography>
+          </Card.HeaderTitle>
+        </Card.Header>
+        <Card.Content>
```

Comment:
It would be better to have the links inside the `Card.Actions` section which will place them at the bottom of the card. 
```suggestion
        <Card.Actions>
```